### PR TITLE
Fix grammar in README

### PR DESCRIPTION
**Pull request by arihant2math**
Fixed grammar mistakes and typos

- Related: #164

### CODE REVIEW

**Comment by onein528** [comment]
@DeveloperWOW64 Please review.

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This is our preferred method. This allows you to always be on the latest version
 
 Released builds can be manually downloaded from this [repository's releases page](https://github.com/FluentHub/FluentHub/releases).
 
-Download the `FluentHub_<versionNumber>.msixbundle` file from the `Assets` section. In order too install the app, you can simply double-click on the .msixbundle file, and the app installer should automatically run. If that fails for any reason, you can try the following command at a PowerShell prompt:
+Download the `FluentHub_<versionNumber>.msixbundle` file from the `Assets` section. In order to install the app, you can simply double-click on the .msixbundle file, and the app installer should automatically run. If that fails for any reason, you can try the following command on a PowerShell prompt:
```
Comment (DeveloperWOW64) [inline README.md:44]:
```diff
- Download the `FluentHub_<versionNumber>.msixbundle` file from the `Assets` section. In order too install the app, you can simply double-click on the .msixbundle file, and the app installer should automatically run. If that fails for any reason, you can try the following command at a PowerShell prompt:
+ Download the `FluentHub_<versionNumber>.msixbundle` file from the `Assets` section. In order to install the app, you can simply double-click on the .msixbundle file, and the app installer should automatically run. If that fails for any reason, you can try the following command on a PowerShell prompt:
```
should use a `with` instead of an `on`

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ If you are interested in fixing issues and contributing directly to the code bas
 
 ### 1. Prerequisites
 
-Ensure you have following components:
+Ensure you have installed following tools:
```
Comment (DeveloperWOW64) [inline README.md:113]:
```diff
- Ensure you have following components:
+ Ensure you have installed following tools:
```
should have a 'the' inbetween the 'following tools' and 'installed' due to the fact that 'tool' is a noun.

File: README.md [hunk 3]
```diff
@@ -147,7 +147,7 @@ See [the documentation](docs/credentials.md).
 
 ## 💳 Credit
 
-- Some application icons created by [Icons8](https://github.com/icons8).
+- Some of the application icons were created by [Icons8](https://github.com/icons8).
```
Comment (DeveloperWOW64) [inline README.md:150]:
```diff
- Some application icons created by [Icons8](https://github.com/icons8).
+ Some of the application icons were created by [Icons8](https://github.com/icons8).
```
The previous was more concise; undo this change.

File: README.md [hunk 4]
```diff
@@ -31,7 +31,7 @@
 
 ### Via Microsoft Store
 
-This is our preferred method. This allows you to always be on the latest version when we release new builds with automatic updates.
+This is the preferred installation method. This allows you to always be on the latest version when we release new builds via automatic updates.
```
Comment (DeveloperWOW64) [inline README.md:34]:
```diff
- This is our preferred method. This allows you to always be on the latest version when we release new builds with automatic updates.
+ This is the preferred installation method. This allows you to always be on the latest version when we release new builds via automatic updates.
```
There's something off about this.. 
Try:
```diff
+ This is the preferred installation method. It allows you to always be on the latest version when we release new builds via automatic updates.
```

**Review by DeveloperWOW64 — CHANGES_REQUESTED** [review]
@arihant2math
*Sorry that this is really fussy*

**Comment by arihant2math** [comment]
@DeveloperWOW64 No worries, I fixed all of it.

File: README.md [hunk 5]
```diff
@@ -147,7 +147,7 @@ See [the documentation](docs/credentials.md).
 
 ## 💳 Credit
 
-- Some of the application icons were created by [Icons8](https://github.com/icons8).
+- Some application icons created by [Icons8](https://github.com/icons8).
```
Comment (DeveloperWOW64) [inline README.md:150]:
```diff
- - Some of the application icons were created by [Icons8](https://github.com/icons8).
+ - Some application icons created by [Icons8](https://github.com/icons8).
```
Could you add a `were` between the `icons` and `created by`?

**Review by DeveloperWOW64 — CHANGES_REQUESTED** [review]
@arihant2math

**Comment by arihant2math** [comment]
@DeveloperWOW64 Fixed that.

**Review by DeveloperWOW64 — APPROVED** [review]
